README.md

```diff
@@ -1,2 +1,127 @@
-# cfd-rust
-WIP (Scheduled for official release by August 31)
+# Crypto Finance Development Kit for Rust (CFD-RUST)
+
+## Dependencies
+
+- Rust
+- C/C++ Compiler
+  - can compile c++11
+  - make support compiler
+- CMake (3.14.3 or higher)
+- Python 3.x
+
+### Windows 
+
+- Visual Studio 2019
```

Comment:
The description was omitted because it is necessary for the windows environment of rust itself.

Like golang, rust basically does a runtime build, so a compiler is required. OSs other than Windows has installed gcc, but since it is not on Windows.
Therefore, install the compiler (vc++).
You can install the compiler with the command:
```
rustup set default-host x86_64-pc-windows-msvc
```

However, in order to use VC++ from the command line, you need to additionally install `Build Tools for Visual Studio 2019`.
https://visualstudio.microsoft.com/downloads/